discovery-provider/plugins/notifications/src/__tests__/mappings/create.test.ts

```diff
@@ -72,6 +74,50 @@ describe('Create Notification', () => {
     })
   })
 
+  test("Process email notification for create tracks", async () => {
+    await createUsers(processor.discoveryDB, [{ user_id: 1 }, { user_id: 2 }])
+    await createSubscription(processor.discoveryDB, [{ subscriber_id: 2, user_id: 1 }])
+    await createTracks(processor.discoveryDB, [{ track_id: 10, owner_id: 1 }, { track_id: 11, owner_id: 1 }])
+    const data: CreateTrackNotification = {
+      track_id: 10,
+    }
+    const data2: CreateTrackNotification = {
+      track_id: 11,
+    }
+
+    const notifications: AppEmailNotification[] = [
+      {
+        type: 'create',
+        timestamp: new Date(),
+        specifier: '1',
+        group_id: 'create:track:user_id:1',
+        data,
+        user_ids: [2],
+        receiver_user_id: 2
+      },
+      {
+        type: 'create',
+        timestamp: new Date(),
+        specifier: '1',
+        group_id: 'create:track:user_id:1',
+        data,
+        user_ids: [2],
+        receiver_user_id: 2
+      }
+    ]
```

Comment:
how do both of these notifs exist? i thought notifications had to have unique group id and specifier?

Reply:
you're right there can't be two notifications with the same group_id and specifier bc there is a multi column unique constraint, but this isn't inserting those notifications into the db - it's just using passing an object into the render method - for the test it doesn't really matter.